DataCapturing/Model/GeoLocation.swift

```diff
@@ -28,10 +28,10 @@ import Foundation
  - SeeAlso: `DataCapturingService`, `PersistenceLayer.save(:[GeoLocation]:MeasurementEntity:(MeasurementMO?, Status) -> Void)`, `GeoLocationMO`
 
  - Author: Klemens Muthmann
- - Version: 1.0.0
+ - Version: 1.1.0
  - Since: 1.0.0
  */
-public class GeoLocation: CustomStringConvertible {
+public struct GeoLocation: CustomStringConvertible {
```

Comment:
I changed this to a struct since a struct provides an automatic constructor for all properties (and is "copy by value" instead of "copy by reference")